app/services/i18n.js

```diff
@@ -15,9 +15,14 @@ export default Service.extend({
 
 		if (fastboot.get('isFastBoot')) {
 			const fs = FastBoot.require('fs');
+			// Per MediaWiki's /languages/messages/MessagesZh_*.php
+			const traditionalChineseLanguages = ['zh-hk', 'zh-mo', 'zh-tw'];
```

Comment:
how about specifying explicitly all Chinese fallbacks in a map to improve readability of code and then L22 change to `[map[language] || language, language.split('-')[0], 'en'].some(lang) => {...}`?

or even we could specify mappings like pt -> pt-br explicitly as well

Reply:
There is no pt -> pt-br, it's the opposite way and I think it's easier to just use `language.split('-')[0]`. The split is also used for languages like zh-cn, zh-sg etc. which fallback to zh.